aten/src/ATen/native/cuda/SoftMax.cu

```diff
@@ -118,8 +118,12 @@ void SpatialSoftMax_getLaunchSizes(
   uint32_t block_threads = block.x * block.y;
   smem_size = block.x == 1 ? 0 : block_threads * sizeof(accscalar_t);
   int max_active_blocks;
+#ifdef __HIP_PLATFORM_HCC__
+  max_active_blocks = 16;
```

Comment:
Where is this HIP limitation documented?

Reply:
it isn't AFAIK, we are working on that.